### ISSUE TITLE
Colour contrast too low

### ISSUE THREAD

**Issue by ConnorTurnbull** [ConnorTurnbull/Project-4#1]
Colour contrast ratio between text and background is low is some areas, making text difficult to read particularly to those with visual impairments.

Ensure that all text on page has a contrast ratio equal to or greater than 4.5:1 for small text and 3:1 for large text, as defined in WCAG 2.0.

**Comment by ConnorTurnbull** [comment]
- Top and bottom hero images text to background contrast ratio does not meet WCAG criteria.
- Changed text colour to white (#FFFFFF) to satisfy WCAG while maintaining site theme.

Before:

![image](https://user-images.githubusercontent.com/110614970/208458286-da60bb56-4395-41cf-899f-378ad7587c4c.png)

![image](https://user-images.githubusercontent.com/110614970/208464071-bf42584f-ae7a-4dbc-9278-134fee60e36c.png)

![image](https://user-images.githubusercontent.com/110614970/208458579-cace9dbc-893e-49b3-b278-d35ac610a6f5.png)

![image](https://user-images.githubusercontent.com/110614970/208460105-1c5a85f2-ffd1-4179-b72c-e1ed4b8c9e49.png)

After:

![image](https://user-images.githubusercontent.com/110614970/208460724-81e55868-27ad-45e3-a58c-2f2bf5be83c8.png)

![image](https://user-images.githubusercontent.com/110614970/208464254-1d7b374b-f649-4e0a-bf2b-e843f2649d1b.png)

![image](https://user-images.githubusercontent.com/110614970/208460646-7b54458c-a3ca-471a-b3f6-f4a5e5d62232.png)

**Comment by ConnorTurnbull** [comment]
- Footer links text to background contrast ratio does not meet WCAG criteria.
- Changed text colour to white (#FFFFFF) to satisfy WCAG while maintaining site theme.
- Also changed footer column headers to white to help with discerning between content. 

Before:

![image](https://user-images.githubusercontent.com/110614970/208467755-8656e057-23e4-4565-9157-59ae9fee2d12.png)

![image](https://user-images.githubusercontent.com/110614970/208503253-66018f1f-db55-4cff-8766-053d3a857dc6.png)


After:

![image](https://user-images.githubusercontent.com/110614970/208470669-c99d7652-5357-4cc6-bd1f-ca22cfcfaf3c.png)

![image](https://user-images.githubusercontent.com/110614970/208470389-3c8158a5-3df6-4b77-919a-d79236258ec9.png)

**Comment by ConnorTurnbull** [comment]
- Changed global font colour from white (rgba 255, 255, 255) to black (#000000) to ensure all content is visible
- Changed anchor styling to black to make visible

Before:

![image](https://user-images.githubusercontent.com/110614970/208478940-37573460-6f79-4fcf-91ce-63c1fe88fe7b.png)

![image](https://user-images.githubusercontent.com/110614970/208479287-0177d7da-13a9-46d5-a8a4-278598f8ace8.png)

After:

![image](https://user-images.githubusercontent.com/110614970/208479352-7a606059-2bda-4a11-9ee9-3d69942812bf.png)

![image](https://user-images.githubusercontent.com/110614970/208479419-27a3084f-99ee-45c6-ada6-11443b265aa1.png)

**Comment by ConnorTurnbull** [comment]
- Keywords showing in header around logo
- Font colour has poor contrast ratio with background

![image](https://user-images.githubusercontent.com/110614970/208523269-930c686e-1932-4891-9dca-5b75fa715bc1.png)

- Issue resolved under #14

**Comment by ConnorTurnbull** [comment]
- Content removed.

![image](https://user-images.githubusercontent.com/110614970/208701928-779ad5bc-8e82-449d-8f6d-a89cccd14e0f.png)

**Comment by ConnorTurnbull** [comment]
- Lighthouse accessibility score improved for 'Home' page.

Before:

![image](https://user-images.githubusercontent.com/110614970/208704611-1c4192ce-fabd-4ff0-930c-4d51f157d6f5.png)

After:

![image](https://user-images.githubusercontent.com/110614970/208704521-58708c6c-8d37-48ca-a9d6-810680b5756c.png)

**Comment by ConnorTurnbull** [comment]
- Contrast issues present on 'Page2'

![image](https://user-images.githubusercontent.com/110614970/208761043-b43b1940-0372-44a3-9bc2-2a95091d0e28.png)

**Comment by ConnorTurnbull** [comment]
Before:

![image](https://user-images.githubusercontent.com/110614970/208761808-f79d1517-064a-4c35-83e4-ff10e1d1dace.png)

![image](https://user-images.githubusercontent.com/110614970/208761739-15c2728f-e67a-4f49-9e46-a80bf7b4cd30.png)

After:

![image](https://user-images.githubusercontent.com/110614970/208769416-a6f49c7e-22dc-4499-a655-48d3cbb7740b.png)

![image](https://user-images.githubusercontent.com/110614970/208769441-708dc6c3-5e64-498a-8afe-353ec598d073.png)

**Comment by ConnorTurnbull** [comment]
- Lighthouse accessibility score improved for 'Page2'

Before:

![image](https://user-images.githubusercontent.com/110614970/208770613-b2625ffb-2de0-46ad-a788-4b16e0aae1d9.png)

After:

![image](https://user-images.githubusercontent.com/110614970/208770567-602062df-11e5-4aca-b290-b2cd0ea4d0ed.png)

**Comment by ConnorTurnbull** [comment]
- Lighthouse reports

[#1 fix - home.pdf](https://github.com/ConnorTurnbull/Project-4/files/10273238/1.fix.-.home.pdf)
[#1 fix - page2.pdf](https://github.com/ConnorTurnbull/Project-4/files/10273239/1.fix.-.page2.pdf)
